src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -809,7 +809,7 @@ private AppPage verifyHtml(By by, String filePath) throws IOException {
 
     private String getPageSource(By by) {
         waitForAjaxLoaderGifToDisappear();
-        String actual = by == null ? getPageSource()
+        String actual = by == null ? browser.driver.findElement(By.tagName("html")).getAttribute("innerHTML")
```

Comment:
Selenium's built-in `getPageSource()` has some abstractions on its own that transforms `disabled=""` and alike to their XHTML counterpart (`disabled="disabled"` etc.), and I don't want that because I want to see the boolean attributes as is.
As for why `innerHTML` instead of `outerHTML`, it's because `outerHTML` will result in:
```html
<html webdriver="true">
  ...
</html>
```